tools/patches/021-add-navbar-bg-vars.patch

```diff
@@ -1,10 +1,10 @@
 diff --git a/inst/lib/bs/scss/_variables.scss b/inst/lib/bs/scss/_variables.scss
-index 7a42c44..aee26ce 100644
+index af39c76..e332bbc 100644
 --- a/inst/lib/bs/scss/_variables.scss
 +++ b/inst/lib/bs/scss/_variables.scss
-@@ -747,6 +747,9 @@ $navbar-toggler-padding-x:          .75rem !default;
- $navbar-toggler-font-size:          $font-size-lg !default;
```

Comment:
`$navbar-toggler-font-size` isn't used anymore?

Reply:
It is https://github.com/rstudio/bslib/pull/254/files#diff-6d46872b8309f0929d61994a2ec3fc406b3dc05588acd0ff002ac58c74bbaabcR747

(It's just no longer included in the patch file)